deploy/ansible/playbooks/group_vars/example/horizon.yml

```diff
@@ -20,3 +20,4 @@ horizon:
           user: stellar
           password: xxx
           host: xxx
+          cursor_name: 'HORIZON'
```

Comment:
@oryband I'd rather this was somehow unique or at least document that it needs to be unique.

Reply:
done